### ISSUE TITLE
Recovery bitcoin core wallet without wallet.dat file... Is it possible?

### ISSUE THREAD

**Issue by theendfear1** [gurnec/btcrecover#155]

**Comment by rterwedo** [comment]
Not possible.  You need data from the wallet (salt, derivations, mkey, and encrypted priv key for that address).

**Comment by theendfear1** [comment]
Hi rterwedo and thanks for your answer.
Could you explain me a little bit better how it works?
I know if i have private key i'm able to unblock the wallet. Isn't the private key generated by the passphrase?  What I'm doing wrong?

**Comment by rterwedo** [comment]
Private key makes public, public makes address.  When you encrypt with a pwd, you encrypt the priv (making an encrypted priv).  

If you have house key, but don't know what door lock to put it in, it does not help.  Additionally you are not even sure what your house key is, so you can't test if its the correct key.

Sorry, I would look for backups etc of the wallet file.

**Comment by theendfear1** [comment]
Ok, but then why this method: 
https://github.com/gurnec/btcrecover/blob/master/docs/Seedrecover_Quick_Start_Guide.md
should instead work with a seed?
What is the difference?

I've read:
"In order to run seedrecover.py, you'll need these two things:
1) A good estimate of what your seed is
2) a receiving address that was generated by your wallet from your seed, along with a good estimate of how many addresses you created before the receiving address you'd like to use"

I'm confused! Forgive my noobity

**Comment by 505Mike** [comment]
I am confused on how to use of seed recovery. I  have all the salt, derivations, mkey, and encrypted priv key for the address. Can i Find my Password from this so i don't risk losing my coins with my wallet.dat

**Comment by rterwedo** [comment]
@505Mike  Yes.  Just make a backup of the wallet.dat or use an extract and you will be fine.

**Comment by nori19870522** [comment]
I am Japanese and do password analysis.
https://keychainx.io/

This is Robert's article.
https://news.bitstarz.com/bitstarz-news-interview-robert-rhodin-keychainx-ceo

I may be able to find the password you lost. Do you need help with your login?

**Comment by Qalander** [comment]
> @505Mike Yes. Just make a backup of the wallet.dat or use an extract and you will be fine.

HI can you show me the process of getting the password from , wallet with mkey and salt, alsong with encripted privt key I have of that wallet.

**Comment by Qalander** [comment]
> Not possible. You need data from the wallet (salt, derivations, mkey, and encrypted priv key for that address).

HI can you show me the process of getting the password from , wallet with mkey and salt, alsong with encripted privte key I have of that wallet.

**Comment by nicoschtein** [comment]
@Qalander Pretty old issue, not sure they'll respond. 
If you have the wallet.dat and you need the password used to unlock it on Bitcoin Core that can be recovered, though it might take longer than a lifetime. Assuming you do have it, without more info on password specs, Id say you still have a chance.

**Comment by btcexchanging** [comment]
if you haven't recovered your wallet, i can help you recover your wallet.
I have recovered most of the common wallets and recovery phrases for many users
you can reach me here at my telegram account:  @walletrecover

**Comment by viladmir2022** [comment]
hi, if you still have wallet problem. I can recover your lost wallet.
I have powerful servers and excellent software that can find any password.
and my fee is only 15-25% of the recovered fund
here is my telegram id: @viladmir2022  and Gmail: walletsrecoveryservice@gmail.com

**Comment by etherx-dev** [comment]
> Hi to everybody!
> 
> I'm trying to recovery bitcoin core wallet (done in 2014) but i don't have the wallet.dat file. I only have the public address and i remember a part of passphrase. Could be this possibile? If yes, can you suggest me the best settings to do it? Thank you so much!

It is possible to scan the hard drive for keys or metadata.

**Comment by Kingcrypto2** [comment]
I have wallet . Data but I forget the password can I recover my btc

**Comment by jasonhoi** [comment]
i can help u i got a method which is much faster than using this library and u can safely send me the code without giving anyone the full private key

**Comment by Kingcrypto2** [comment]
> i can help u i got a method which is much faster than using this library and u can safely send me the code without giving anyone the full private key

Ok where I send u

**Comment by 505Mike** [comment]
I still have my wallet can't remember password any help would be greatly appreciated

**Comment by jueduishiren** [comment]
> I still have my wallet can't remember password any help would be greatly appreciated

test all the possibilities of the PWs you have in mind.
